index.qmd

```diff
@@ -14,9 +14,9 @@ At [Openscapes](https://openscapes.org), we think about open science like a land
 ```
 ## Reflecting together
 
-The motivation for Openscapes Reflections is to provide a space to reflect on your workflows and plan how open data science practices can improve your daily habits and fuel institutional culture change.
+Openscapes Reflections provides a space to reflect on your workflows and plan how open data science practices can improve your daily habits and fuel institutional culture change.
 
-The Reflections Program is set up to be an affordable, self-paced way to learn and incorporate open data science principles into your workflow, no matter where you are starting from. Through weekly prompts and community huddles, we will reflect together through a process that leaves you feeling empowered and intentional about your work and how you can get it done. We're hoping this is something that could be valuable for you year after year, both for reflecting yourself and onboarding your team. We designed this to complement to our ongoing initiatives, particularly the [Openscapes Champions Program](https://openscapes.org/champions), so here you can explore your workflows in a self-driven manner aligned with the Pathway document.
+The Reflections Program is set up to be an affordable, self-paced way to learn and incorporate open data science principles into your workflow, no matter where you are starting from. <!--# preceding sentence largely redundant with first sentence. What's different is "affordable, self-paced ... no matter where you are starting from -->Through weekly prompts and community huddles, we will reflect together through a process that leaves you feeling empowered and intentional about your work and how you can get it done. We're hoping this is something that will be valuable for you year after year, both for reflecting yourself and onboarding your team. We designed this to complement our ongoing initiatives, particularly the [Openscapes Champions Program](https://openscapes.org/champions), so here you can explore your workflows in a self-driven manner aligned with the Pathway document<!--# ref to Pathway doc comes out of nowhwere -->.
```

Comment:
Thank you! I've provided a workflow example right here and deleted the redundancy